tests/python/unittest/test_tir_schedule_compute_inline.py

```diff
@@ -622,8 +621,8 @@ def elementwise_overcomputed_producer_reverse_inlined(
     for i, j in T.grid(128, 128):
         with T.block("B"):
             vi, vj = T.axis.remap("SS", [i, j])
-            T.where(i < 127 and j < 127)
-            C[vi, vj] = A[vi, vj] * 2.0 + 1.0
+            if vi < 127 and vj < 127:
```

Comment:
What's the reason to change `T.where` to `IF` branches? There are two major concerns:
1. with `If` branch we may lose changes for further schedule opportunities after `compute_inline`
2. It's breaking changes for the existing codebase

Reply:
To be very clear, T.where limits the iteration domain, meaning it’s going to change the affinity of iteration space, which is not desirable. This PR fixed this behavior and I count it as a bug fix rather than a breaking change.

Reply:
We had some discussions on this topic. T.where is a mechanism to use to handle the iterator predicate outside the block. We had a restriction that the affine map indicated by T.where plus the axis binding must match the loop domain declared.

In the specific case of reverse compute inline with over relaxing part, we are not exactly changing the loop domain, but change the internal operations, since we are iterating over the loops and would like to skip some operations. So having if here is better and matches the design of block isolation